koswat/configuration/io/config_sections/config_section_fom_base.py

```diff
@@ -0,0 +1,74 @@
+import abc
+import math
+from configparser import SectionProxy
+from typing import Any, Optional
+
+from koswat.configuration.settings.koswat_general_settings import SurtaxFactorEnum
+from koswat.core.io.file_object_model_protocol import FileObjectModelProtocol
+
+
+class ConfigSectionFomBase(FileObjectModelProtocol, abc.ABC):
+    """
+    Base class for configuration section file object models.
+    """
+
+    _surtax_mappings: dict
+    _float_mappings: dict
```

Comment:
I would not advocate for this but rather an abstract get method:

```suggestion
    @abc.abstractmethod
    def get_surtax_mappings(self) -> dict:
      ...
    @abc.abstractmethod
    def get_float_mappings(self) -> dict:
      ...
```